cloudevents/pydantic/event.py

```diff
@@ -0,0 +1,327 @@
+#  Copyright 2018-Present The CloudEvents Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+import datetime
+import typing
+
+import pydantic
+
+from cloudevents import http
+from cloudevents.http import dict_methods
+from cloudevents.http.dict_methods import encode_attribute_value
+from cloudevents.sdk.event.attributes import (
+    id_attribute,
+    specversion_attribute,
+    time_attribute,
+)
+
+
+def _ce_json_dumps(obj: typing.Dict[str, typing.Any], *args, **kwargs) -> str:  # noqa
+    return http.to_json(dict_methods.from_dict(obj)).decode("utf-8")
+
+
+def _ce_json_loads(
+    data: typing.Union[str, bytes], *args, **kwargs  # noqa
+) -> typing.Dict[typing.Any, typing.Any]:
+    return dict_methods.to_dict(http.from_json(data))
```

Comment:
Why noqa here?

Please add docs. And I'd probably rather have intermediate variables than one-liners. Just for the clarity.